dienst2/settings.py

```diff
@@ -203,4 +214,6 @@
     'horizontal_field_class': 'col-md-8',
 }
 
+STATICFILES_STORAGE = 'whitenoise.storage.CompressedManifestStaticFilesStorage'
```

Comment:
I feel this has some overlap with what `django-compressor` already does. If this is indeed the case, please change it, otherwise I'm fine with the PR :)